rules/hardcoded_credentials.go

```diff
@@ -98,11 +112,29 @@ func (r *credentials) matchValueSpec(valueSpec *ast.ValueSpec, ctx *gosec.Contex
 			}
 		}
 	}
+
+	// Match values
+	for _, ident := range valueSpec.Values {
+		if val, err := gosec.GetString(ident); err == nil {
+			if r.patternValue.MatchString(val) {
+				if r.ignoreEntropy || (!r.ignoreEntropy && r.isHighEntropyString(val)) {
+					return ctx.NewIssue(valueSpec, r.ID(), r.What, r.Severity, r.Confidence), nil
+				}
+			}
+		}
+	}
+
 	return nil, nil
 }
 
 func (r *credentials) matchEqualityCheck(binaryExpr *ast.BinaryExpr, ctx *gosec.Context) (*issue.Issue, error) {
 	if binaryExpr.Op == token.EQL || binaryExpr.Op == token.NEQ {
+		identStrConst, okLit := binaryExpr.X.(*ast.BasicLit)
```

Comment:
Looks like the rest of the code base uses `ok`. Unless there's a good reason to specify this in particular, it'd be good to match the codebase

Reply:
I did this because there were two ok variables, but I noticed I could rearrange the code to resolve this.